tests/controllers/test_response_correlation_controller.py

```diff
@@ -8,19 +8,19 @@
 )
 
 from webviz_ert.controllers.response_correlation_controller import (
-    sort_dataframe,
+    _sort_dataframe,
 )
 
 
-def test_sort_dataframe():
+def test__sort_dataframe():
```

Comment:
Different than the tests for the other internal functions, where I (not sure if we) used only one `_`. Do you think we should change all of them to be consistent though the test file?

Reply:
ah, this was just a result of blind replacing / renaming. gonna fix it in this instance, don't think we need a double underscore, it's just a name after all : )

and done.